src/sessions/VirtualPageLoadTimer.ts

```diff
@@ -63,6 +64,10 @@ export class VirtualPageLoadTimer extends MonkeyPatched {
         document.addEventListener('keydown', this.updateLatestInteractionTime);
     }
 
+    load?(context: PluginContext): void {
+        throw new Error('Method not implemented.');
```

Comment:
nit: I'd prefer if this function did nothing rather than throwing an error to follow a fail-safe convention.

Reply:
Yup -- my ide created this fn (as it said the abstract method was not implemented)

I can make it empty